tests/functional/test_k4_three_layer_composite_geometric.py

```
@@ -54,6 +55,35 @@ def _planted_ciphertext() -> tuple[str, dict]:
+        summary = run_three_layer_composite_geometric(
+            subst_alphabets={"KRYPTOS": KNOWN_KEYED_ALPHABETS["KRYPTOS"]},
+            order_names=["row_major"],
+            reflection_names=["identity"],
+            rotation_offsets=[0],
+            remainder_modes=["trailing"],
+            priority_clock_times=BERLIN_WALL_PRIORITY_TIMES,
+            null_artifact_path=tmp_path / "null.json",
+            graph_path=tmp_path / "graph.json",
+            eureka_snapshot_path=tmp_path / "snap.md",
+        )
+        assert summary["run_params"]["clock_states_tested"] == BERLIN_WALL_PRIORITY_TIMES
```

Pylint (4.0.7)

[convention] Line too long (89/88)

(C0301)